### PR TITLE
Bump python to 3.7

### PR DESCRIPTION
**Pull request by SmithPeder**
**TL;DR UPDATE**: I built this image locally and pushed to dockerhub under the `:test` tag, and it doesn't really work 🔥I don't think `make` is located in the `-alpine` image
https://ci.abakus.no/webkom/lego/2785/5

**TL;DR UPDATE 2** I buildt the image without alpine and pushed it as `:python3.7`. We are trying to get [this branch](https://github.com/webkom/lego/pull/1555) to pass with the new python version. And the [last run](https://ci.abakus.no/webkom/lego/2794) passed like a charm

The 'real' tox image is running 3.7 with alpine, so we should be able to do the same https://hub.docker.com/r/younata/tox

image

### CODE REVIEW

**Review by odinuge — APPROVED** [review]
lgtm :kr: 

Just don't merge before this is properly tested (as I mentioned here: https://github.com/webkom/lego/pull/1555) :smile: 

PS2: And squash commits :1st_place_medal:
